parser/openfoodfacts_taxonomy_parser/parser/parser.py

```diff
@@ -83,7 +83,8 @@ def _create_entry_nodes(self, entry_nodes: list[NodeData], project_label: str):
             id: entry_node.id,
             preceding_lines: entry_node.preceding_lines,
             src_position: entry_node.src_position,
-            main_language: entry_node.main_language
+            main_language: entry_node.main_language,
+            is_external: entry_node.is_external
```

Comment:
sorry for this change, but I think it would be important to remember which taxonomy an entry comes from.

So that if we display the message that an entry is external, the user knows where to edit it.

So either:
* keep is_external and add an external_taxonomy property
* change is_external to external: with this property being empty meaning it's internal.

It could also be leverage in search to find entries from a specific external taxonomy !